kolibri_instant_schools_plugin/assets/src/views/sign-up-page/index.vue

```diff
@@ -4,7 +4,7 @@
 
     <ui-toolbar type="colored" textColor="white">
       <template slot="icon">
-        <ui-icon class="app-bar-icon"><logo/></ui-icon>
+        <img src="../img/instant-school-logo.png" class="app-bar-icon">
```

Comment:
Also, let's just use the logo component here too?

Reply:
Per discussion, ignoring this comment